pages/mesosphere/dcos/services/data-science-engine/1.0.1/upgrade/index.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: layout.pug
+navigationTitle: Upgrade 
+excerpt: Upgrading DC/OS Data Science Engine
+title: Upgrade
+menuWeight: 7
+enterprise: true
+model: /mesosphere/dcos/services/data-science-engine/data.yml
+render: mustache
+---
+
+{{ model.techName }} does not support in-place upgrades. In order to transition to a new version you must manually back up your data located in your `jupyter_data` folder and upload it to a new instance of {{ model.techName }}, running on the latest version using the following guide.
+
+1. Navigate to the root of your Jupyter Notebook UI and run the following command to compress the contents of your jupyter_data folder:
```

Comment:
should we mention that this should be done using Terminal in JupyterLab?

Reply:
Done